content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -671,6 +652,38 @@ Each feature gate is designed for enabling/disabling a specific feature:
   to `IngressClass.spec.parameters`.
 - `Initializers`: Allow asynchronous coordination of object creation using the
   Initializers admission plugin.
+- `InTreePluginAWSUnregister`: Stops registering the EBS in-tree plugin in
```

Comment:
The InTreePluginXXXUnregister is not only a rename. This is to decouple unregistering a plugin from CSI migration. Previously if you want to disable an intree plugin. There is no flag that can do it. The only way is to enable CSI migration and enable this Complete flag.

Now with the new flag. You can set it directly to disable an intree plugin without enabling CSI migration.

See https://github.com/kubernetes/website/pull/28223 for more context.